.github/workflows/build-linux-clang-x64-dynamic.yml

```
@@ -33,10 +36,43 @@ jobs:
+        cd data
+        wget https://github.com/Open-Cascade-SAS/OCCT/releases/download/V7_8_0/opencascade-dataset-7.8.0.tar.xz
+        tar -xf opencascade-dataset-7.8.0.tar.xz
+
+    - name: Install Xvfb
+      run: sudo apt-get install -y xvfb
+
+    - name: Start Xvfb
+      run: Xvfb :99 -screen 0 1920x1080x24 &
+
+    - name: Set DISPLAY environment variable
+      run: echo "DISPLAY=:99" >> $GITHUB_ENV
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:23: Double quote to prevent globbing and word splitting

(shellcheck)